tests/parametric/test_span_links.py

```diff
@@ -123,15 +142,19 @@ def test_span_link_from_distributed_w3c_headers(self, test_agent, test_library):
         with test_library:
             with test_library.start_span(
                 "root",
-                links=[Link(parent_id=0)],
                 http_headers=[
                     ["traceparent", "00-12345678901234567890123456789012-1234567890123456-01"],
                     ["tracestate", "foo=1,dd=t.dm:-4;s:2,bar=baz"],
                 ],
             ):
                 pass
 
-        span = get_span(test_agent)
+            with test_library.start_span("span"):
+                pass
+
+        traces = test_agent.wait_for_num_traces(2)
+        root = traces[0][0]
+        span = traces[1][0]
```

Comment:
Same question here: Why the second span is supposed to have link?

Reply:
I will update this test to make it more clear